docs/apps/components/collapse.md

```diff
@@ -0,0 +1,124 @@
+5. **Nested Collapse**: Organize complex forms with multiple levels of collapsible sections.
+6. **Enhanced animations and transitions**: Improve the visual appeal of the collapse/expand action.
+7. **Accessibility considerations**: Optimize the collapse component for users with disabilities.
+
+## API
+
+### Methods
+
+| **Name**| **Description**|**Parameters**|
+|---------|----------------|--------------|
+|`this.addClassName`|Add style class to form item|`(fields: String(componentID), String[]; className: String)`|
+|`this.removeClassName`|Remove form item style|`(fields: String(componentID), String[]; className: String)`|
+|`this.getComponent`|Returns a component whose id has been passed as a parameter|`(component_ID: String(componentID), String[]; Object: Object)`|
+|`this.hide`|Hides the field|`(fields: String(componentID)`|
+|`this.show`|Displays the field|`(fields: String(componentID)`|
+|`this.disable`| Disable input field from user interaction|`(fields: String(componentID), String[])`|
+|`this.getValue`|Get A Value From a component|`(fieldName String(componentID)`|
+
+!!! Info
+    1. The show() and hide() methods can also be used to control the visibility of an alert in response to user input.
+    2. **fields** refer to the component **ID**. You can fetch the ID from the **Component Attribute** panel in the Page Builder.
+    3. For using **this.show(fields)** enable the **Hidden** check-box in the **Component Attribute** panel.
+
+#### Steps to place/use the methods for components of the Page Builder"
+
+1. Go to Form Attribute :material-menu-right: Action Panel :material-menu-right: Setting :material-menu-right: (Mounted | refresh | click 'Add action')
+2. Write the method/code as shown in the **Example** below
+3. Click on `Save`.
+4. On the main screen click on `Save` again.
+5. Click on `Preview` to see the code in action.
+<img src= "/apps/components/img/alert1.png">
+
+!!! Example
+    1. `this.addClassName(fields, className)`
+          * Go to Form Attribute :material-menu-right: Style Sheets :material-menu-right: add the class
+            ```js
+            .abc{ // abc is the class name
+            background-color: red;
+            }
+            ```
+          * Follow the steps in the above **Note**
+          * ```js
+            this.addClassName('collapse_itxbeisp', 'abc')
+            ```
+
+    2. `this.removeClassName(fields, className)`
+        ```js
+        this.removeClassName('collapse_itxbeisp', 'abc')
+        ```
+    
+    3. `this.hide(fields)`
+        ```js
+        var fields= ['collapse_itxbeisp']
+        this.hide(fields)
+        ```
+    
+    4. `this.show(fields)`
+        ```js
+        var fields= ['collapse_itxbeisp']
+        this.show(fields)
+        ```
+    
+    5. `this.disable(['fields'])`
+        ```js
+        this.disable(['collapse_itxbeisp'])
+        ```
+    
+    6. `this.getValue('fieldName')`
```

Comment:
This function returns undefined, therefore lacks use on this component. Like I suggested earlier that it should be removed.

Reply:
removed